.github/workflows/release.yml

```
@@ -0,0 +1,77 @@
+name: Release
+# Controls when the workflow will run
+on:
+    # Triggers the workflow on push or pull request events but only for the master branch
+    schedule:
+      - cron: "0 8 * * Sun"
+on:
```

actionlint (1.7.7)

key "on" is duplicated in workflow. previously defined at line:3,col:1

(syntax-check)